pages/common/multitail.md

```diff
@@ -0,0 +1,19 @@
+# multitail
+
+> Monitor last part of multiple files in multiple windows in a terminal.
+
+- Monitor two files in two separate windows:
+
+`multitail {{file1}} {{file2}}`
+
+- Merge two files in one window:
+
+`multitail {{file1}} -I {{file2}}`
+
+- Show two files in two windows side by side:
+
+`multitail -s {{file1}} {{file2}}`
+
+- Show output of two commands:
+
+`multitail -l "ping {{host1}}" -l "ping {{host1}}"`
```

Comment:
Actually, the entire command should be in `{{}}`. So maybe write something like - `multitail -l "{{echo hello}}" -l "{{echo world}}"`

Also - what is the difference between this command and simply doing `echo hello && echo world` ?

Reply:
`ping` continuously adds data, so `multitail -l "{{ping host1}}" -l "{{ping host2}}"` will keep adding lines for both commands in parallel. So what about:

```
- Keep showing appended output of two commands:
```

Reply:
I like - 'Follow the interleaved output from two commands:'. I think `follow` is a better term which explains what is going on, rather than `show appended output` which stutters a bit.